salt/search/processes.sls

```diff
@@ -20,10 +20,12 @@ search-processes-task:
             {% endfor %}
 
 search-processes-start:
-    cmd.run:
-        - name: start search-processes
+    service.running:
```

Comment:
misleading as `search-processes` is a task ([script that runs and exits](https://github.com/elifesciences/builder-base-formula/blob/master/elife/config/etc-init-multiple-processes-parallel.conf)), not a service; so this will always be detected as stopped and be re-executed.
Systemd has better separation of services and tasks.

Reply:
if it runs and then exits then it's just plain wrong. I'll revert it

Reply:
actually, not much to revert. closing